src/main/java/org/openelisglobal/ocl/OclZipImporter.java

```diff
@@ -0,0 +1,131 @@
+package org.openelisglobal.integration.ocl;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OclZipImporter {
+    private static final Logger log = LoggerFactory.getLogger(OclZipImporter.class);
+
+    @Value("${org.openelisglobal.ocl.zip.path}")
+    private String oclZipPath;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private final CsvMapper csvMapper = new CsvMapper();
+
+    /**
+     * Imports and parses the OCL ZIP package, returning the parsed data. Also logs
+     * the complete JSON output for inspection.
+     * 
+     * @return List of JsonNode objects representing parsed JSON/CSV files.
+     * @throws IOException if the import fails.
+     */
+    public List<JsonNode> importOclZip() throws IOException {
```

Comment:
Hello @vsvishalsharma definately we cant have a separate method just for logiignd info.
You can still log with in `importOclPackage`